dipy/tracking/utils.py

```diff
@@ -1048,3 +1047,47 @@ def min_radius_curvature_from_angle(max_angle, step_size):
         max_angle = np.pi / 2.0
     min_radius_curvature = step_size / 2 / np.sin(max_angle / 2)
     return min_radius_curvature
+
+
+def seeds_directions_pairs(positions, peaks, max_cross=-1):
```

Comment:
We start to introduce keyword-only arguments in the codebase
Can you replace this by:

`def seeds_directions_pairs(positions, peaks, *, max_cross=-1)`

Reply:
done.